examples/ImageJImgLib2.ipynb

```diff
@@ -31,24 +33,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Added 357 JARs to the Java classpath.\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "# Initialize imagej. Point to a local Fiji installation\n",
+    "# Replace the value below with to your local Fiji installation.\n",
     "fiji_path = '/home/matt/bin/Fiji.app/'\n",
     "\n",
     "# Note: order is important!\n",
     "import imagej\n",
-    "imagej.quiet_init(fiji_path)\n",
+    "imagej.init(fiji_path)\n",
```

Comment:
In addition to changing the method to `init`, I also made it create an instance of `net.imagej.ImageJ` internally, so that you can simply write:
```
ij = init(fiji_path)
```
Without all the `import imglyb` etc. rigamarole. (Remember how the order of those imports was super important to get right, or else things didn't work? Shouldn't be an issue anymore.)

Now you can just import `imglyb` the first time you need one of the conversion methods. And only import `autoclass` if/when you need to wrap additional Java classes.

Reply:
That's fantastic!

Here's a follow-up:

  https://github.com/InsightSoftwareConsortium/itk-jupyter-widgets/pull/38